ethvm/main.cpp

```diff
@@ -83,10 +84,11 @@ int main(int argc, char** argv)
 	Address sender = Address(69);
 	Address origin = Address(69);
 	u256 value = 0;
-	u256 gas = state.gasLimitRemaining();
+	u256 gas = 0;
```

Comment:
This initializer should not matter at all. Using Block().gasLimitRemaining() is not allowed, and at this point there is no Block object avaiable. This vaule is the remaining gas in a block, it makes no sense to initialize it with any value fdifferent from zero before we have started to build a blockchain.